src/digisos/skjema/familie/forsorgerplikt/ForsorgerPlikt.tsx

```diff
@@ -62,6 +66,7 @@ const ForsorgerPliktView = () => {
                 <FormattedHTMLMessage id="familierelasjon.ingress" values={{antallBarn}} />
                 <SysteminfoMedSkjema>
                     <RegistrerteBarn />
+                    {/*<BrukerregistrerteBarn/>*/}
```

Comment:
Kanskje en liten kommentar for hvorfor den er kommentert ut?
`Disablet fram til alle fagsystem kan håndtere brukerregistrerte barn.`